CONTRIBUTING.md

```diff
@@ -121,6 +121,10 @@ the work.
 in [Required Approvals](#required-approvals)), a Committer other than the PR
 contributor may squash and merge changes into the master branch.
 
+See also (from the OCIO Developer Guide):
+* [Getting started](http://opencolorio.org/developers/geting_started.html)
```

Comment:
typo, should be http://opencolorio.org/developers/getting_started.html two 't'

Reply:
Good catch. Fixed.